www-client/thorium-browser-bin/thorium-browser-bin-124.0.6367.218.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CHROMIUM_LANGS="af am ar bg bn ca cs da de el en-GB en-US es es-419 et fa fi fil fr
+	gu he hi hr hu id it ja kn ko lt lv ml mr ms nb nl pl pt-BR pt-PT ro ru sk
+	sl sr sv sw ta te th tr uk ur vi zh-CN zh-TW"
+
+inherit desktop xdg unpacker
+
+DESCRIPTION="Thorium Browser: A fast and secure browser for the modern web."
+HOMEPAGE="https://github.com/Alex313031/thorium"
+
+IUSE="+avx2 sse3"
+
+SRC_URI="avx2? ( https://github.com/Alex313031/thorium/releases/download/M${PV}/thorium-browser_${PV}_AVX2.deb )
+         sse3? ( https://github.com/Alex313031/thorium/releases/download/M${PV}/thorium-browser_${PV}_SSE3.deb )"
+
+RESTRICT="strip"
+LICENSE="BSD-3-Clause-Thorium"
```

Comment:
As I understand, we store licenses in https://github.com/gentoo/guru/tree/master/licenses

Reply:
Just moved it